core/cocos2d.h

```diff
@@ -60,10 +60,12 @@ namespace cocos2d = ax;
 #define CC_ASSERT                    AX_ASSERT
 #define CC_CONSTRUCTOR_ACCESS        public
 
+#ifndef AX_DISABLE_DEPRECATED
```

Comment:
Maybe everything in this header should be disabled by `AX_DISABLE_DEPRECATED`?

Reply:
Many references from extensions, and refactor 3rd extensions is time-wasting